pkgs/applications/networking/remote/remote-desktop-manager/default.nix

```
@@ -0,0 +1,79 @@
+{ stdenv
+, lib
+, fetchurl
+, dpkg
+, autoPatchelfHook
+, libXext
+, libxcrypt-legacy
+, libSM
+, cups
+, vte
+, kerberos
+, lttng-ust_2_12
+, webkitgtk
+, libsoup
+, icu
+, openssl
+, makeWrapper
+, gtk3-x11
+,
+}:
+let
+  pname = "remote-desktop-manager";
+  version = "2023.3.1.4";
+in
+stdenv.mkDerivation {
+  inherit pname version;
+
+  src = fetchurl {
+    url = "https://cdn.devolutions.net/download/Linux/RDM/${version}/RemoteDesktopManager_${version}_amd64.deb";
+    sha256 = "sha256-XTPDZP01j8PNmvMqynlRiaXOc0QX+ZSWtehQWc/X70Q=";
+  };
+
+  nativeBuildInputs = [
+    dpkg
+    autoPatchelfHook
+    makeWrapper
+  ];
+
+  buildInputs = [
+    libXext
+    libxcrypt-legacy
+    libSM
+    cups
+    vte
+    kerberos
+    lttng-ust_2_12
+    webkitgtk
+    libsoup
+    stdenv.cc.cc.lib
+  ];
+
+  unpackPhase = "dpkg-deb -x $src .";
+
+  installPhase = ''
+    mkdir $out
+    cp -r bin usr $out
+
+    ln -s $out/usr/share $out/share
```

Can this be moved instead of symlinked? Will it break the program?
```suggestion
    mv $out/usr/share $out/share
    rmdir $out/usr
```